tests/test_wcs_utils.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+import kbmod.analysis.wcs_utils
+import astropy.coordinates
+import astropy.units
+
+
+class test_construct_wcs_tangent_projection(unittest.TestCase):
+    def test_requires_parameters(self):
+        with self.assertRaises(TypeError):
```

Comment:
I do not see where the function raises a `TypeError`. Is that from the lack of the first (positional) parameter?

Reply:
Yes. the first parameter is required and so python helpfully throws an exception.